bftengine/src/preprocessor/tests/messages/PreProcessResultMsg_test.cpp

```diff
@@ -49,6 +49,7 @@ bftEngine::impl::SigManager* createSigManagerWithSigning(
                           replicasKeysFormat,
                           publicKeysOfClients,
                           concord::crypto::KeyFormat::HexaDecimalStrippedFormat,
+                          {0, "", concord::crypto::KeyFormat::PemFormat},
```

Comment:
you don't need this change if the parameter is optional.

Reply:
done